wallet/src/actors/worker/error.rs

```diff
@@ -44,6 +44,8 @@ pub enum Error {
         _0, _1
     )]
     NodeBehindLocalTip(u32, u32),
+    #[fail(display = "provided birth_date epoch is greater than the current: {}", _0)]
```

Comment:
Sorry, it may be ambiguous to whether that is the provided or the current one 

```suggestion
    #[fail(display = "provided birth_date epoch ({}) is greater than the current", _0)]
```